src/pipeline/NodeBindings.cpp

```diff
@@ -986,9 +1033,9 @@ void NodeBindings::bind(pybind11::module& m, void* pCallstack){
 
     // DetectionNetwork node
     detectionNetwork
-        .def_readonly("input", &DetectionNetwork::input, DOC(dai, node, DetectionNetwork, input))
-        .def_readonly("out", &DetectionNetwork::out, DOC(dai, node, DetectionNetwork, out))
-        .def_readonly("passthrough", &DetectionNetwork::passthrough, DOC(dai, node, DetectionNetwork, passthrough))
+        .def_readonly("input", &DetectionNetwork::input, DOC(dai, node, NeuralNetwork, input))
+        .def_readonly("out", &DetectionNetwork::out, DOC(dai, node, NeuralNetwork, out))
```

Comment:
`out` should be `DOC(dai, node, DetectionNetwork, out))`, since it overrides `NeuralNetwork`'s output.